cmd/filter/pathfilter.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package filter
+
+import (
+	"strings"
+
+	"github.com/openebs/node-disk-manager/cmd/controller"
+	"github.com/openebs/node-disk-manager/pkg/util"
+)
+
+const (
+	pathFilterKey = "path-filter"
+)
+
+var (
+	pathFilterName  = "path filter"  // filter device paths
+	pathFilterState = defaultEnabled // filter state
+	includePaths    = ""
```

Comment:
Can we add default exclude loop here if ndm is not able to read from config map then there will be a problem.

Reply:
in what scenarios will we not have the config map? Having said that.. it is good idea to add loop as a default filter. Will add it.

Reply:
added loop!